src/libraries/System.Diagnostics.TraceSource/src/System.Diagnostics.TraceSource.csproj

```diff
@@ -32,22 +32,6 @@
     <Compile Include="$(CommonPath)System\Diagnostics\TraceListenerHelpers.cs"
```

Comment:
TraceListenerHelpers.cs does not seem to be used by this library anymore

Reply:
It's still used here:
https://github.com/dotnet/runtime/pull/38908/files#diff-8cc735e882562064167814fed669b96dR41
but if I just delete GetThreadId and use CurrentManagedThreadId in its place as you suggested above, this will go away, anyway.